src/Http/Resources/CP/Concerns/HasRequestedColumns.php

```diff
@@ -23,7 +23,7 @@ protected function visibleColumns()
 
         return collect($requested)
             ->filter(fn ($field) => $columns->has($field))
-            ->map(fn ($field) => $columns->get($field)->visible(true))
+            ->map(fn ($field) => $columns->get($field)?->visible(true))
```

Comment:
Are you sure this is fixing whatever the problem is?

On the line before it is filtering out any columns that aren't there.

Reply:
You're right. 

It has been fixed via https://github.com/statamic/cms/pull/5915

We are simply on a older version. The next update will fix it for us.

Reply:
Ah, great!